streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractTask.java

```diff
@@ -110,24 +110,21 @@ public final ThreadCache cache() {
         return cache;
     }
 
+    public abstract void init();
+    public abstract void resume();
```

Comment:
`init` and `resume` are not used in standby tasks at all, so maybe we can just remove them from the abstract tasks (see my comments below)?